script.js

```diff
@@ -0,0 +1,128 @@
+console.log("hello world");
+
+window.addEventListener("DOMContentLoaded", createQuiz);
+
+function createQuiz(){
+
+const questions = [
+  { id : 1,
+    question: "Which of the following is a primary component of Good Manufacturing Practice (GMP) regulations?",
+    answers: [
+            { text: "Maintaining personal hygiene in the workplace", correct: true },
+            { text: "Reducing production costs to increase profit margins", correct: false },
+            { text: "Increasing the number of employees to speed up production", correct: false},
+            { text: "Using the cheapest available raw materials", correct: false },
```

Comment:
When a key/value pair is a boolean then it is common practice to name it `isCorrect` or `isFalse`. It's not incorrect what you've done but it makes it easier to understand in the code itself for yourself and future collaborators.

Reply:
Great!! I will keep this in mind, thanks